libs/core/src/lib/card/card-footer.component.html

```diff
@@ -0,0 +1,6 @@
+<ng-content></ng-content>
+<div class="fd-card__footer-actions" *ngIf="actionItems?.length > 0">
+    <div class="fd-card__footer-actions-item" *ngFor="let actionItem of actionItems">
```

Comment:
Can we do `<ng-content select="fdCardFooterActionItem">`? And apply the `fd-card__footer-actions-item` class in the host of the `CardFooterActionItemDirective` directive. Then we would not need the AfterViewInit in `CardFooter`

Reply:
But then how to decide if actions div should exist or not?